app_sidebar_collapsible.py

```diff
@@ -29,15 +30,17 @@
 import flask_talisman as flt
 
 
-
-OPENPATH_LOGO = "https://www.nrel.gov/transportation/assets/images/openpath-logo.jpg"
+OPENPATH_LOGO = os.path.join(os.getcwd(), "assets/openpath-logo.jpg")
+encoded_image = base64.b64encode(open(OPENPATH_LOGO, 'rb').read()).decode("utf-8")
```

Comment:
This is clearly correct, but just checking on why we didn't want to keep using the public openpath logo file?

Reply:
Per your commit [comment here ](https://github.nrel.gov/nrel-cloud-computing/nrelopenpath-admin-dashboard/commit/47f023dc1d6b5a6531693a90e0575830691356ec)and the issue opened [here](https://github.com/e-mission/op-admin-dashboard/issues/43#issue-1718938898):

> Loading [the logo] from the NREL website is bad because:
> 
> * it introduces a cross-site dependency
> * it introduces extra cookies from the NREL site that are not required for our codebase
> 
> I have removed the OpenPATH logo for now; we should reintroduce it, load it from the local filesystem and add a favicon

Hence, that issue can be closed upon this merge 😄